### PR TITLE
Add LICENSE

### PR DESCRIPTION
**Pull request by lxylxy123456**
Same as nbgrader

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Project Jupyter Contributors
+Copyright (c) 2020 Project Jupyter Contributors
```
Comment (aalmanza1998) [inline LICENSE]: Should we say Team KALE instead of Project Jupyter Contributers?

Comment (lxylxy123456) [reply LICENSE]: I mentioned this in the hangouts message. We need all 4 group members and client's feedback.

**Comment by aalmanza1998** [comment]
Professor Nitta suggested using Project Jupyter Contributers.

**Comment by moorepants** [comment]
This depends on who wrote the code. If you copied some existing code and modified it you would often modify the license copyright statement like so:

```
Copyright (c) 2020 Jason Moore
Copyright (c) 2017-2019 Suzy Q.
```

Did you all copy anyone else's code here?

If you all wrote it, the copyright belongs to whoever wrote the code.

An aside, are you planning to add to nbgrader, hubshare, or any existing project? How does this relate to the other exisitng projects?

**Comment by lxylxy123456** [comment]
I believe this repo only contains our own code (need to check with Kevin). 
This repo serves as a backend of our project, and we are going to modify nbgrader code in this repo as frontend: https://github.com/rkevin-arch/nbgrader-k8s

**Comment by moorepants** [comment]
If you all wrote this code then I'd start by listing your names explicitly as the copyright holders on that copyright line. Each real name should be listed.

**Comment by moorepants** [comment]
You can read some about this in various stackexchange postings, like:

https://opensource.stackexchange.com/questions/886/copyright-and-contributing-to-an-open-source-project

https://opensource.stackexchange.com/questions/886/copyright-and-contributing-to-an-open-source-project

Or here: https://www.linuxfoundation.org/blog/2020/01/copyright-notices-in-open-source-software-projects/

**Comment by moorepants** [comment]
Maybe following the last link it should be "the ngshare contributors". You can always keep and AUTHORS file to list the actual contributors.

**Pull request merged**